src/main/java/io/openliberty/tools/common/plugins/util/PrepareFeatureUtil.java

```diff
@@ -100,11 +101,12 @@ private void prepareFeature(String groupId, String artifactId, String version) t
 			json = downloadArtifact(groupId, FEATURES_JSON_ARTIFACT_ID, "json", version);
 		} catch (PluginExecutionException e) {
 			debug(e);
-			info(String.format("Cannot find json for coordinate %s:%s:%s in connected repositories", groupId,
+			info(String.format("Features.json was not found at the coordinate %s:"+FEATURES_JSON_ARTIFACT_ID+":%s in connected repositories", groupId,
```

Comment:
Is the file really capital `Features.json`? Perhaps "The features.json file was not found at the expected coordinate.....". Also end the message with a `.`.